src/lib/wapi.js

```diff
@@ -1541,8 +1530,8 @@ window.WAPI.sendImage = async function (imgBase64, chatid, filename, caption, qu
  * @param newName - string the new name to set as profile name
  */
 window.WAPI.setMyName = async function (newName) {
-    if(!Store.Versions.default[11].BinaryProtocol) Store.Versions.default[11].BinaryProtocol=new Store.bp(11);
-    return await Store.Versions.default[11].setPushname(newName);
+  if (!Store.Base.BinaryProtocol) Store.Base.setSubProtocol(11);
+  return await Store.Base.setPushname(newName);
 }
```

Comment:
This new bp set up is not working as there is no more Store.Base in new version of WAPI.